src/main/java/codeu/model/store/basic/MessageStore.java

```diff
@@ -105,5 +109,14 @@ public void addMessage(Message message) {
   /** Sets the List of Messages stored by this MessageStore. */
   public void setMessages(List<Message> messages) {
     this.messages = messages;
+    addIndivMessages(messages);
+  }
+  
+  private void addIndivMessages(List<Message> messages) {
```

Comment:
This would probably be better as

private void addMessageTouser(Message message) {
  // do thing.
}

then in setMessages, you can iterate over messages and call addMessageToUser.

The goal is to reduce code duplication in the getAuthorId to user to adding message step.